recipes/parastation/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -ex
+
+unset FFLAGS F77 F90 F95
+
+export CC=$(basename "$CC")
+export CXX=$(basename "$CXX")
+export FC=$(basename "$FC")
+
+cd $SRC_DIR/psmpi
+
+./autogen.sh
+
+mkdir -p build
+cd build
+
+../configure --prefix=$PREFIX \
+             --with-confset=gcc \
+             --enable-confset-overwrite \
+             --with-pscom-allin=$SRC_DIR/pscom \
+             --with-hwloc=$PREFIX \
+             --with-pmix=$PREFIX \
+             --enable-msa-awareness \
+             --enable-threading
+
+make -j"${CPU_COUNT}"
+
```

Comment:
Is there a `make check` we can run here?

Reply:
This version of ParaStation MPI is configured with PMIx and not Hydra (these configure options are mutually exclusive), hence no mpirun/mpiexec is available and `make test` can't run conventionally without an MPI launcher.
However, `mpichversion` was added to verify the ParaStationMPI installation as well as an "Hello World MPI" example in Fortran for compilation testing.